src/metadata/taglib_handler.cc

```diff
@@ -416,12 +416,16 @@ void TagLibHandler::extractMP3(TagLib::IOStream* roStream, const std::shared_ptr
             if (frameList.isEmpty())
                 continue;
 
-            const TagLib::ID3v2::Frame* frame = frameList.front();
-            const auto textFrame = dynamic_cast<const TagLib::ID3v2::TextIdentificationFrame*>(frame);
-
-            const TagLib::String frameContents = textFrame->toString();
-            std::string value(frameContents.toCString(true));
-            value = sc->convert(value);
+            std::string value;
+            for (const auto& frame : frameList) {
+                const auto textFrame = dynamic_cast<const TagLib::ID3v2::TextIdentificationFrame*>(frame);
+                const TagLib::String frameContents = textFrame->toString();
+                std::string fvalue(frameContents.toCString(true));
+                fvalue = sc->convert(fvalue);
```

Comment:
I would just do nested call. No need to come up with names for temporaries... But it's a matter of taste since no function makes use of rvalues.